### DISCUSSION TITLE
ragflow chatbot to postman connection

### DISCUSSION THREAD

**Discussion by Naren-NS** [infiniflow/ragflow#2644] [Q&A]
Is there a way to send the query in the JSON body of the request via Postman and receive a response from the chatbot that gets displayed in Postman?

**Comment by Feiue** [comment]
Add Bearer authorization in the Authorization header,
 the URI is `api/v1/session/completion`,
 and the body format is 
{
"session_id":"your_session_id",
"question":"your_question"
}.
 For authorization details, refer to the article [api](https://github.com/infiniflow/ragflow/blob/main/docs/references/api.md). 
And there is another api in [api](https://github.com/infiniflow/ragflow/blob/main/docs/references/api.md), you can also use it.

**Reply by Naren-NS** [reply]
Hey thanks for the response, its working okay, but the issue is that i have a very large query(the value for the "content" key). And also, the chatbot responds with something like "...\nFor the content length reason, it stopped, continue?" for a query which is 4-5 lines too.

Any suggestions would help.
![Screenshot 2024-10-07 121220](https://github.com/user-attachments/assets/3b59bd84-4af4-456a-b614-43eb2ce40736)

**Reply by Feiue** [reply]
> Hey thanks for the response, its working okay, but the issue is that i have a very large query(the value for the "content" key). And also, the chatbot responds with something like "...\nFor the content length reason, it stopped, continue?" for a query which is 4-5 lines too.
> 
> Any suggestions would help. ![Screenshot 2024-10-07 121220](https://private-user-images.githubusercontent.com/106026507/374057958-3b59bd84-4af4-456a-b614-43eb2ce40736.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MjgzODQyNTgsIm5iZiI6MTcyODM4Mzk1OCwicGF0aCI6Ii8xMDYwMjY1MDcvMzc0MDU3OTU4LTNiNTliZDg0LTRhZjQtNDU2YS1iNjE0LTQzZWIyY2U0MDczNi5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQxMDA4JTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MTAwOFQxMDM5MThaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT1lNGNkN2EyMTM2MjdkYWE0YTMxYTNkMTdhNDI4M2EwMjVjYjg2ZjQyOGI4ZGZlMTMwMDIzNTcwYmUxZWY4ZTRhJlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCJ9.HvDJkYcFeJF2i55XkO8Ip2YEU_xNhHFWzFWzDE3MqCM)

You can set max_tokens for the dialog that the conversation belongs to.

**Comment by Naren-NS** [comment]
Did that but didn't work, when I changed the model it worked.
